website/static/js/contribAdder.js

```diff
@@ -80,6 +79,26 @@ var AddContributorViewModel = oop.extend(Paginator, {
             return self.query() && !self.results().length;
         });
 
+        self.addAllVisible = ko.pureComputed(function() {
+            var selected_ids = self.selection().map(function(user){
+                return user.id;
+            });
+            for(var i = 0; i < self.results().length; i++) {
+                if(self.contributors().indexOf(self.results()[i].id) === -1 &&
```

Comment:
This can be simplified to `return self.contributors().indexOf(self.results()[i].id) === -1 && selected_ids.indexOf(self.results()[i].id) === -1)`